### PR TITLE
Convert all usages of MarkdownSpan -> UnsafeRenderedMarkdown

### PR DESCRIPTION
**Pull request by joshlory**
Most places, the difference between inline and block didn't make a huge difference, so I didn't worry about removing the extra 10px bottom padding that is added via CSS to all `<p>` tags.

### `detail_view_contents.jsx` before:

Screen Shot 2019-04-02 at 3 48 43 PM

### `detail_view_contents.jsx` after:

Screen Shot 2019-04-02 at 3 47 55 PM

### `detail_view_response.jsx` before:

Screen Shot 2019-04-02 at 3 41 56 PM

### `detail_view_response.jsx` after:

Screen Shot 2019-04-02 at 3 40 39 PM

However the `LabeledFormComponent.jsx` depended on the markdown being inline, so I added a wrapper div with `display: inline-block`.

Screen Shot 2019-04-02 at 3 59 10 PM

### CODE REVIEW

File: apps/src/templates/UnsafeRenderedMarkdown.jsx [hunk 1]
```diff
@@ -6,9 +6,12 @@ import Parser from '@code-dot-org/redactable-markdown';
 import expandableImages from './plugins/expandableImages';
 import xmlAsTopLevelBlock from './plugins/xmlAsTopLevelBlock';
 import stripStyles from './plugins/stripStyles';
+import externalLinks from 'remark-external-links';
 
 const remarkParser = Parser.create();
-remarkParser.parser.use([xmlAsTopLevelBlock, expandableImages]);
+remarkParser.parser
+  .use([xmlAsTopLevelBlock, expandableImages])
+  .use(externalLinks, {target: '_blank', rel: false});
```
Comment (Hamms) [inline apps/src/templates/UnsafeRenderedMarkdown.jsx]: we _always_ want to use `rel='noreferrer noopener'` when setting `target=_blank`.

See https://mathiasbynens.github.io/rel-noopener/

**Review by Hamms — COMMENTED** [review]
Nice!

I'm super concerned about applying target=_blank globally, though, particularly because after glancing at that library it looks like by "external links" it actually means "any anchor using an absolute (rather than relative) url", which I believe will apply to a lot of our _internal_ links as well.

Can we try to identify the cases here in which we _actually_ do want this functionality, and only enable it for those?

**Comment by joshlory** [comment]
Right, sorry! I tested this case, but forgot to add the screenshot. What I found was any `code.org` link or `*.code.org` link was seen as "internal".

Screen Shot 2019-04-02 at 12 46 37 PM

One weird edge case: it seems to treat `#foo` anchor links as external — I don't think we do this in markdown, so I'm not going to stress too much about this edge case.

**Comment by joshlory** [comment]
Ah never mind, https://www.npmjs.com/package/is-absolute-url doesn't take the current domain name into account 😕. I'll fix.

**Comment by joshlory** [comment]
@Hamms this should be ready for review now!

**Review by Hamms — APPROVED** [review]
:tada: LGTM! :tada:

**Pull request merged**